docs/visualize/tilemap.asciidoc

```diff
@@ -77,6 +77,10 @@ based on the geohash coordinates.
 
 NOTE: By default, the *Change precision on map zoom* box is checked. Uncheck the box to disable this behavior.
 
+NOTE: By default, the *place markers off grid (use geocentroid) box is checked. When this box is checked, the markers are
+placed in the center of all the documents in that bucket. When it is not checked, the markers are placed in the center
```

Comment:
"placed in the center of all the documents in that bucket" - can we make this sentence more specific to the geo_points or location in those documents? For example, "placed in the center of all the documents geo_points in that bucket"